.eslintrc

```diff
@@ -1,6 +1,11 @@
 {
-  "extends": ["airbnb-typescript", "plugin:prettier/recommended"],
+  "extends": [
+    "airbnb-typescript",
```

Comment:
Small suggestions: 

While we are at it, what do you think of adding [next.js eslint plugin](https://nextjs.org/docs/basic-features/eslint)?

Every time we start the web app, it displays a warning saying we should probably use this.

(We can do it in another PR if you prefer)

Reply:
Hmm, why not! They are interesting extras 👀 

I have opened a dedicated issue since I doubt that i'd be straightforward to enable: https://github.com/labelflow/labelflow/issues/661